.github/workflows/deployFreighterApiProduction.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy @stellar/freighter-api
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Release Number
+        default: "2.0.0"
+        required: true
+jobs:
+  bump-version:
+    name: Checkout code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: release/${{ github.event.inputs.release }}
+      - name: Build package
+        uses: actions/setup-node@v4
+        with:
+          node-version: "21"
+      - name: Version npm package
+        run: |
+          cd @stellar/freighter-api
+          yarn version --new-version ${{ github.event.inputs.release }}
```

Comment:
this command bumps the version and automatically commits it to the local codebase, so all we need to is open a PR next